WordPress-Phoenix/wordpress-development-toolkit
# BHGC-1685: unit test out of the box

## Description

Adding unit test out of the box

## Testing Instruction
Generate a copy of the plugin with this as base, 
Instruction are set here.
https://github.com/WordPress-Phoenix/wordpress-development-toolkit

![Screen Shot 2019-04-24 at 2 55 21 PM](https://user-images.githubusercontent.com/2894340/56685917-00298680-66a1-11e9-9718-95975045b2f0.png)

Then run unit test via 
```
./bin/phpunit --coverage-clover clover.xml --config ./.circleci/phpunit.xml
./bin/coverage-check clover.xml 80
```

![Screen Shot 2019-04-24 at 2 58 16 PM](https://user-images.githubusercontent.com/2894340/56686151-6c0bef00-66a1-11e9-9388-873973ba3035.png)

## Supplement Screens
The auto-generated file that was produced
<img width="1021" alt="Screen Shot 2019-04-24 at 3 00 07 PM" src="https://user-images.githubusercontent.com/2894340/56686353-cd33c280-66a1-11e9-9b42-ad14ed14e3a6.png">

<img width="1128" alt="Screen Shot 2019-04-24 at 3 00 33 PM" src="https://user-images.githubusercontent.com/2894340/56686365-d0c74980-66a1-11e9-9f29-9060faf0274f.png">

<img width="1116" alt="Screen Shot 2019-04-24 at 3 00 46 PM" src="https://user-images.githubusercontent.com/2894340/56686366-d0c74980-66a1-11e9-96be-7702e1293b81.png">
